apps/web/vercel.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://openapi.vercel.sh/vercel.json",
+  "git": {
+    "deploymentEnabled": {
+      "main": false
+    }
+  },
+  "ignoreCommand": "git diff --quiet HEAD^ HEAD ../../{patches,package.json,turbo.json} ../../packages/{colors,components,icons,status-js} ./",
```

Comment:
non-blocking: is `../../packages/{colors,components,icons,status-js}` all required?

Reply:
side not, turbo repo ignore

Reply:
> is ... required

in other words, why bother